### DISCUSSION TITLE
Tests with the deformation of the armature

### DISCUSSION THREAD

**Discussion by ghost** [UPBGE/upbge#1721] [Resources for UPBGE 0.3+]
We already know that armor deformation has several performance issues in version 3.0, but testing it we discovered something strange, and we wanted to let you know, maybe it will be useful in solving the problem.

If we import the same model with the same armature and animation from UPBGE 2.5 and then change the deform to BGE, then copy it through the shortcut control+c and control+v in UPBGE 3.0, the armature problems disappear, the FPS is stable and has the same performance as 2.5, we found this very strange..

When we tested with Normal Map the FPS went down again, but we already expected that because we already knew the problem, what intrigued us was to achieve the performance if copied and pasted directly from one version to the other, in the end, I hope it helps to solve the problem.

**Comment by josephajtaX** [comment]
That Strange it does not work for me
